fastly/provider.go

```diff
@@ -51,32 +51,34 @@ func Provider() *schema.Provider {
-			"fastly_ngwaf_thresholds":             dataSourceFastlyNGWAFThresholds(),
-			"fastly_ngwaf_redactions":             dataSourceFastlyNGWAFRedactions(),
-			"fastly_ngwaf_workspaces":             dataSourceFastlyNGWAFWorkspaces(),
-			"fastly_package_hash":                 dataSourceFastlyPackageHash(),
-			"fastly_secretstores":                 dataSourceFastlySecretStores(),
-			"fastly_services":                     dataSourceFastlyServices(),
-			"fastly_tls_activation":               dataSourceFastlyTLSActivation(),
-			"fastly_tls_activation_ids":           dataSourceFastlyTLSActivationIDs(),
-			"fastly_tls_certificate":              dataSourceFastlyTLSCertificate(),
-			"fastly_tls_certificate_ids":          dataSourceFastlyTLSCertificateIDs(),
-			"fastly_tls_configuration":            dataSourceFastlyTLSConfiguration(),
-			"fastly_tls_configuration_ids":        dataSourceFastlyTLSConfigurationIDs(),
-			"fastly_tls_domain":                   dataSourceFastlyTLSDomain(),
-			"fastly_tls_platform_certificate":     dataSourceFastlyTLSPlatformCertificate(),
-			"fastly_tls_platform_certificate_ids": dataSourceFastlyTLSPlatformCertificateIDs(),
-			"fastly_tls_private_key":              dataSourceFastlyTLSPrivateKey(),
-			"fastly_tls_private_key_ids":          dataSourceFastlyTLSPrivateKeyIDs(),
-			"fastly_tls_subscription":             dataSourceFastlyTLSSubscription(),
-			"fastly_tls_subscription_ids":         dataSourceFastlyTLSSubscriptionIDs(),
-			"fastly_vcl_snippets":                 dataSourceFastlyVCLSnippets(),
+			"fastly_configstores":                    dataSourceFastlyConfigStores(),
+			"fastly_datacenters":                     dataSourceFastlyDatacenters(),
+			"fastly_dictionaries":                    dataSourceFastlyDictionaries(),
+			"fastly_ip_ranges":                       dataSourceFastlyIPRanges(),
+			"fastly_kvstores":                        dataSourceFastlyKVStores(),
+			"fastly_ngwaf_alert_datadog_integration": dataSourceFastlyNGWAFAlertDatadogIntegration(),
+			"fastly_ngwaf_alert_jira_integration":    dataSourceFastlyNGWAFAlertJiraIntegration(),
+			"fastly_ngwaf_redactions":                dataSourceFastlyNGWAFRedactions(),
+			"fastly_ngwaf_thresholds":                dataSourceFastlyNGWAFThresholds(),
+			"fastly_ngwaf_virtual_patches":           dataSourceFastlyNGWAFVirtualPatches(),
+			"fastly_ngwaf_workspaces":                dataSourceFastlyNGWAFWorkspaces(),
+			"fastly_package_hash":                    dataSourceFastlyPackageHash(),
+			"fastly_secretstores":                    dataSourceFastlySecretStores(),
+			"fastly_services":                        dataSourceFastlyServices(),
+			"fastly_tls_activation":                  dataSourceFastlyTLSActivation(),
+			"fastly_tls_activation_ids":              dataSourceFastlyTLSActivationIDs(),
+			"fastly_tls_certificate":                 dataSourceFastlyTLSCertificate(),
```

Comment:
Gotta love Go standard formatting, where the spacing between columns is dependent on the length of the name in the first column...